File: python/prism_sdk/local_discovery_rate_integrity_contract_model.py
"""Metrics P32 local contract_model discovery-rate integrity feature F05."""
from .discovery_rate_integrity_support import DiscoveryRateRequest4,DiscoveryRateCard7,DiscoveryRateIntegrityError,manifest,qualify
FEATURE_ID="AFA-metrics-P32-F05";CONTRACT_VERSION="metrics-local_discovery_rate_integrity_contract_model/1.0"
def local_discovery_rate_integrity_contract_model_manifest():return manifest(feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="local",mode="contract_model")
def qualify_local_discovery_rate_integrity_contract_model(request:DiscoveryRateRequest4)->DiscoveryRateCard7:return qualify(request,feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="local",mode="contract_model")

File: crates/metrics/src/local_discovery_rate_integrity_contract_model.rs
//! Metrics P32 local contract_model discovery-rate integrity feature.
use super::discovery_rate_integrity_support::{manifest,qualify,DiscoveryRateCard7,DiscoveryRateIntegrityError,DiscoveryRateRequest4};
pub const FEATURE_ID:&str="AFA-metrics-P32-F05";pub const CONTRACT_VERSION:&str="metrics-local_discovery_rate_integrity_contract_model/1.0";
pub fn local_discovery_rate_integrity_contract_model_manifest()->serde_json::Value{manifest(FEATURE_ID,CONTRACT_VERSION,"local","contract_model")}
pub fn qualify_local_discovery_rate_integrity_contract_model(request:&DiscoveryRateRequest4)->Result<DiscoveryRateCard7,DiscoveryRateIntegrityError>{qualify(request,FEATURE_ID,CONTRACT_VERSION,"local","contract_model")}
